### ISSUE TITLE
add datetime validator

### ISSUE THREAD

**Issue by sbatschelet** [racke/Data-Transpose#16]
If we could add DateTime::Format::DateManip as a dep sth like below could cover DT validation.

```
my $dt = DateTime::Format::DateManip->parse_datetime( $date ); 
```
if you could add me as a contributor I will write is up.  also open to other validation ideas but I think this is useful.

**Comment by racke** [comment]
A DateTime validator would be indeed useful, but I don't see why we need to add a complete different DateTime Perl module.

**Comment by sbatschelet** [comment]
From DateTime POD This module does not parse dates! 
what about DateTime::Format::ISO8601?

**Comment by racke** [comment]
On 07/29/2015 12:02 AM, Sam Batschelet (hexfusion) wrote:
> From DateTime POD This module does not parse dates!
> what about DateTime::Format::ISO8601?
> 

Yes, please use this.

Regards
	Racke

-- 
Perl and Dancer Development

Visit our Perl::Dancer conference 2015.
More information on http://www.perl.dance.

**Comment by racke** [comment]
Closed by PR #17 and #18.
